db/migrate/20230610133458_add_auth_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthTokenToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :auth_token, :string 
```

Comment:
since we are using `auth_token` attribute to find User, there should be an index for it for fast searching and it should be unique so that no two users have same auth_token index